backend/todo/models.py

```diff
@@ -1,10 +1,21 @@
 from tkinter import CASCADE
 from django.db import models
 from django.conf import settings
-from datetime import datetime    
-
+from datetime import datetime 
 # Create your models here.
 
+class Todo(models.Model):
```

Comment:
this looks the same as `class BasicReminderInfo`
unless there's a difference i'm not seeing, inherit from that and delete this `Todo` class